miri

```diff
@@ -154,6 +159,9 @@ find_sysroot() {
 
 # Run command.
 case "$COMMAND" in
+toolchain)
+    "$MIRIDIR/rustup-toolchain" "$@"
+    ;;
```

Comment:
When I said "move it up to the other commands" I meant the ones around line 70. :)